### ISSUE TITLE
SORTA download link broken

### ISSUE THREAD

**Issue by dennishendriksen** [molgenis/molgenis#5541]
#### Reproduce
- Import [biobank_ontology.owl.zip](https://github.com/molgenis/molgenis/files/591833/biobank_ontology.owl.zip) using the 'Upload' plugin
- Select the 'SORTA' plugin in the 'Data integration' menu
- Select 'New task'
- Set the name 'sorta test' for the new task
- Paste the content of [sorta_test.txt](https://github.com/molgenis/molgenis/files/591839/sorta_test.txt) in the text area
- Select 'Match'
- Wait until the matching has completed
- Select 'Go to result' for the job with name 'sorta test'
- Select 'Download'

#### Expected
- CSV download starts

#### Observed
```
This site can’t be reached

The webpage at https://molgenis62.gcc.rug.nl/menu/dataintegration/sortaservice/match/download/AAAACWDH2XZIIATAPVHJR3AAAM?threshold= might be temporarily down or it may have moved permanently to a new web address.
ERR_INVALID_RESPONSE
```

**Comment by dennishendriksen** [comment]
Might be fixed by @ChaoPang in https://github.com/ChaoPang/molgenis/commit/e9d8f1e8086b075d6b49b2050bb69fddf68f4e3f

**Comment by dennishendriksen** [comment]
Fixed in #6720
